builds/runtimes/pypy-5.7.1

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Build Path: /app/.heroku/python/
+# Build Deps: libraries/sqlite
+
+# NOTICE: This formula only works for the cedar-14 stack, not cedar.
+
+OUT_PREFIX=$1
+
+echo "Building PyPy..."
+SOURCE_TARBALL='https://bitbucket.org/pypy/pypy/downloads/pypy2-v5.7.1-linux64.tar.bz2'
+curl -L $SOURCE_TARBALL | tar jx
+cp -R pypy2-v5.7.0-linux64/* $OUT_PREFIX
```

Comment:
Line 10 says v5.7.1 and line 12 says v5.7.0.

Perhaps the best (DRY) solution would be to put `PYPY_VERSION="pypy2-v5.7.1"` on line 8 that then use `${PYPY_VERSION}` on lines 9, 10, and 12.